### PR TITLE
LPS-146857 Update places where small empty space should appear (data-engine)

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
Related Issue: https://issues.liferay.com/browse/LPS-146857

Hi, team!
I was working on an echo issue ([LPS-146857](https://issues.liferay.com/browse/LPS-146857)) where we propose to update the places where the small Empty State should appear and we have seen that one of those places is in the Web Content structures, specifically in the Builder Search, which belongs to the data engine module (data-engine-taglib). So we have looked for a solution and we would like to know if you agree with it for the update.

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @BarbaraCabrera, can you please check if these failures are related to your changes? Thanks

**Comment by BarbaraCabrera** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-types/FieldTypeList.es.js [hunk 1]
```diff
@@ -97,7 +96,16 @@ const FieldTypeList = ({
...
+		return (
+			<ClayEmptyState
+				description={Liferay.Language.get(
+					'try-again-with-a-different-search'
+				)}
+				imgSrc={`${themeDisplay.getPathThemeImages()}/states/search_state.gif`}
+				small
+				title={Liferay.Language.get('no-results-found')}
+			/>
+		);
```
Comment (carolmariaabb) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-types/FieldTypeList.es.js:108]: Hi @BarbaraCabrera,
Do we really need to change the messages? Is it a change decided by a designer?

**Comment by liferay-forms** [comment]
Hey @BarbaraCabrera, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:test:forms

**Comment by carolmariaabb** [comment]
@BarbaraCabrera please fix the conflicts so I can forward this PR.

Thanks!

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-sets/FieldSetList.js [hunk 2]
```diff
@@ -156,19 +156,13 @@ export default function FieldSetList({searchTerm}) {
...
-							),
-						}}
-						keywords={searchTerm}
+					<ClayEmptyState
+						description={Liferay.Language.get(
+							'try-again-with-a-different-search'
+						)}
+						imgSrc={`${themeDisplay.getPathThemeImages()}/states/search_state.gif`}
 						small
+						title={Liferay.Language.get('no-results-found')}
```
[the remark this answers is not in the document]
Comment (BarbaraCabrera) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-sets/FieldSetList.js]: Hi! Sorry for the delay. I have been talking with the designers of my team and we have seen that it is better to leave it as it was but unify using empty states with the prop small.
Thanks for your feedback!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @BarbaraCabrera, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @BarbaraCabrera, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:reevaluate:1377401_1382

**Comment by liferay-forms** [comment]
Hey @BarbaraCabrera, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**